CMakeLists.txt

```diff
@@ -610,6 +610,20 @@ if(USE_ASAN)
     string(APPEND CMAKE_LINKER_FLAGS_DEBUG " -fsanitize=address")
 endif()
 
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
+  include(CheckCSourceCompiles)
+  check_c_source_compiles("#include <arm_neon.h>
+int main() {
+  float a[] = {1.0, 1.0};
+  vst1q_f32_x2(a);
```

Comment:
This is missing a second parameter: `vst1q_f32_x2 (float32_t * __a, float32x4x2_t val)`

Reply:
this code should do it:
```
float32x4x2_t v;
v.val[0] = vcombine_f32 (vcreate_f32 (__AARCH64_UINT64_C (0)), vcreate_f32 (__AARCH64_UINT64_C (0)));
v.val[1] = vcombine_f32 (vcreate_f32 (__AARCH64_UINT64_C (0)), vcreate_f32 (__AARCH64_UINT64_C (0)));
vst1q_f32_x2(a, v);
```